api/provider/aws/service_create.go

```diff
@@ -60,12 +60,18 @@ func (s *ServiceProvider) Create(req models.CreateServiceRequest) (string, error
 	var loadBalancerRole string
 	if req.LoadBalancerID != "" {
 		fqLoadBalancerID := addLayer0Prefix(s.Config.Instance(), req.LoadBalancerID)
-		loadBalancerDescription, err := describeLoadBalancer(s.AWS.ELB, fqLoadBalancerID)
+		lb, err := describeLoadBalancer(s.AWS.ELB, s.AWS.ALB, fqLoadBalancerID)
 		if err != nil {
 			return "", err
 		}
 
-		if aws.StringValue(loadBalancerDescription.Scheme) == "internet-facing" {
+		if lb.isALB && req.Stateful {
+			errorMsg := "application load balancer with a stateful service is not currently" +
```

Comment:
[nit] the pattern in this codebase is to use `text` and multi-line assignment, e.g.:
```
text := "application ... "
text += "supported; please ...."
return fmt.Errorf(text)
```

Reply:
✅